assets/eip-5773/contracts/utils/MultiAssetRenderUtils.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Are you unable to re-license this as `CC0-1.0`?

Reply:
It just slipped through the cracks, I updated it in the latest commit.